tox.ini

```diff
@@ -1,6 +1,7 @@
 [tox]
 minversion = 2.0
 envlist = docs,flake8,mypy,coverage,py{27,34,35,36,py},du{11,12,13,14}
+skipsdist = True
```

Comment:
This looks very good!

Reply:
Aye, it speeds up tox initialization massively and it was totally unnecessary, as noted in the commit. Win-win!